contracts/base-minter/src/contract.rs

```diff
@@ -42,6 +41,9 @@ pub fn instantiate(
 
     let factory = info.sender.clone();
 
+    // set default status so it can be queried without failing
+    STATUS.save(deps.storage, &Status::default())?;
```

Comment:
is default of bool `false`?

Reply:
Yeah the default is `false`. But since the `STATUS` storage is in the contract, the item has to be saved at least once on init to exist and not fail `.load()` queries.